tests/yast2_gui/yast2_control_center.pm

```diff
@@ -204,8 +204,6 @@ sub start_partitioner {
 sub start_vpn_gateway {
     search('vpn');
     assert_and_click 'yast2_control-center_vpn-gateway-client';
-    record_soft_failure('bsc#1191112 - Resizing window as workaround for YaST content not loading');
-    send_key_until_needlematch('yast2-vpn-gateway-client', 'alt-f10', 20, 9);
```

Comment:
still missing needles in several places in this file.

Reply:
I checked the PR added the workaround for this file, it just added these two lines, need I assert this needle?

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/cae6400303ef59c58de143fbaf3050bf5e22457e/tests/yast2_gui/yast2_control_center.pm#L207

Reply:
please check that commit to apply below to the rest of the cases.

Reply:
Updated code.